mongo/client_encryption.go

```diff
@@ -73,8 +73,9 @@ func NewClientEncryption(keyVaultClient *Client, opts ...*options.ClientEncrypti
 	return ce, nil
 }
 
-// CreateEncryptedCollection creates a new collection with the help of automatic generation of new encryption data keys for null keyIds.
+// CreateEncryptedCollection creates a new collection for Queryable Encryption with the help of automatic generation of new encryption data keys for null keyIds.
 // It returns the created collection and the encrypted fields document used to create it.
+// Queryable Encryption is in Public Technical Preview. Queryable Encryption should not be used in production and is subject to backwards breaking changes.
```

Comment:
[Optional] I understand this is how we commented in [GODRIVER-2449](https://jira.mongodb.org/browse/GODRIVER-2449), but can we mark "Beta: ..." here as [DRIVERS-2349](https://jira.mongodb.org/browse/DRIVERS-2349) suggests though it is not a Golang convention as "Deprecated: ..."?

Reply:
Done. Also updated Range algorithm notes with "Beta:".

Reply:
👍